llvm/test/Transforms/LoopVectorize/reduction-inloop-uf4.ll

```diff
@@ -66,6 +66,88 @@ entry:
   ret i32 %sum.0.lcssa
 }
 
+define i64 @reduction_sum_double(ptr noalias %p, ptr noalias %q) {
```

Comment:
Could you add a brief comment here to say that this checks 2 reductions chained together or update the name? Not sure if `_double` makes this entirely clear